--- mcpp-2.7.2/configure.ac.bero	2013-01-16 01:13:48.746887270 +0100
+++ mcpp-2.7.2/configure.ac	2013-01-16 01:14:02.570722662 +0100
@@ -6,7 +6,7 @@ dnl To cross-compile you must edit here
 
 AC_INIT( mcpp, 2.7.2, kmatsui@t3.rim.or.jp)
 AC_CONFIG_SRCDIR( src/main.c)
-AM_CONFIG_HEADER( src/config.h)
+AC_CONFIG_HEADERS( [src/config.h])
 AC_CONFIG_AUX_DIR( config)
 AC_CANONICAL_TARGET
 AM_INIT_AUTOMAKE( no-dependencies)
